MekHQ/src/mekhq/campaign/finances/Lease.java

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright (c) 2009 Jay Lawson (jaylawson39 at yahoo.com). All rights reserved.
+ * Copyright (C) 2013-2025 The MegaMek Team. All Rights Reserved.
+ *
+ * This file is part of MekHQ.
+ *
+ * MekHQ is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License (GPL),
+ * version 3 or (at your option) any later version,
+ * as published by the Free Software Foundation.
+ *
+ * MekHQ is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty
+ * of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * A copy of the GPL should have been included with this project;
+ * if not, see <https://www.gnu.org/licenses/>.
+ *
+ * NOTICE: The MegaMek organization is a non-profit group of volunteers
+ * creating free software for the BattleTech community.
+ *
+ * MechWarrior, BattleMech, `Mech and AeroTech are registered trademarks
+ * of The Topps Company, Inc. All Rights Reserved.
+ *
+ * Catalyst Game Labs and the Catalyst Game Labs logo are trademarks of
+ * InMediaRes Productions, LLC.
+ */
+package mekhq.campaign.finances;
+
+import java.io.PrintWriter;
+import java.time.LocalDate;
+
+import megamek.common.Entity;
+import megamek.logging.MMLogger;
+import mekhq.campaign.unit.Unit;
+import mekhq.campaign.unit.UnitOrder;
+import mekhq.utilities.MHQXMLUtility;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+
+public class Lease {
+    private static final MMLogger logger = MMLogger.create(UnitOrder.class);
+
+    private Money leaseCost;
+    private LocalDate acquisitionDate;
+
+    /**
+     * Leases are nominally attached to units while they are in the hanger.
+     */
+    public Lease(LocalDate currentDay, Unit unit) {
+        acquisitionDate = currentDay;
+        // Campaign Operations 4th., p43
+        leaseCost = unit.getSellValue().multipliedBy(0.005);
```

Comment:
Magic number need to be a constant.
It's also important to put somewhere the limit to what types of units are available for lease, as the leasing for transport actually only applies for dropships and transport ships, not for meks, ASF, etc.